io/net/bonding.py.data/bonding.yaml

```diff
@@ -0,0 +1,9 @@
+Test: !mux
+    option:
+        bonding_mode: "0 1 2 3 4 5 6"
+Parameters:
+    Iface1: "enP3p1s0f0"
+    Iface2: "enP3p1s0f1"
+    peerip: "85.85.85.85"
+    peerif2: "eth3"
```

Comment:
It is a case of scalability vs speed.
The present method can only handle 2 interfaces in host and 2 in peer.
Suggested method can handle any number of interfaces in host and peer, as specified in multiplexer file.

For your specific problem, try creating bonding with mode 0 in peer once, and create all 6 modes in host at a stretch. This way, you configure bonding 7 times (1 time on peer + 6 times in host)